src/platform/tls_mitls.c

```diff
@@ -21,7 +21,7 @@
 
 uint16_t QuicTlsTPHeaderSize = 0;
 
-#define QUIC_SUPPORTED_CIPHER_SUITES        "TLS_AES_128_GCM_SHA256"
+#define QUIC_SUPPORTED_CIPHER_SUITES        "TLS_AES_128_GCM_SHA256:TLS_CHACHA20_POLY1305_SHA256"
```

Comment:
We should match the same everywhere. Prefix this with `TLS_AES_256_GCM_SHA384`?

Reply:
Does mitls support that?

Reply:
It _should_.